### PR TITLE
Tweak up flaking end-to-end tests of API Priority and Fairness

### LINKED ISSUE #96710: "highqps" not reaching 95% completion threshold on API priority and fairness ... , blocking gce-master-default
**Which jobs are flaking**:
gce-cos-master-default

**Which test(s) are flaking**:
Kubernetes e2e suite.[sig-api-machinery] API priority and fairness should ensure that requests can't be drowned out (priority)

**Testgrid link**:
https://testgrid.k8s.io/sig-release-master-blocking#gce-cos-master-default

**Reason for failure**:

```
/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/apimachinery/flowcontrol.go:76
Nov 18 22:04:59.326: client "highqps": got 94.6% completed requests, want at least 95%
/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/vendor/github.com/onsi/ginkgo/internal/leafnodes/runner.go:113
```

**Anything else we need to know**:
- go.k8s.io/triage at present only has data up to 11/11/2020
Three failures here 
https://prow.k8s.io/view/gcs/kubernetes-jenkins/logs/ci-kubernetes-e2e-gci-gce/1329227465779843072
https://prow.k8s.io/view/gcs/kubernetes-jenkins/logs/ci-kubernetes-e2e-gci-gce/1329209095974031360
https://prow.k8s.io/view/gcs/kubernetes-jenkins/logs/ci-kubernetes-e2e-gci-gce/1329180404124487680

### PR DESCRIPTION
**Pull request by MikeSpreitzer**
**What type of PR is this?**

/kind bug
/kind flake

**What this PR does / why we need it**:
This builds on #96798 by reducing the expected throughput, since the main problem seems to be that the expected throughput is sometimes not achieved.

**Which issue(s) this PR fixes**:

Fixes #96710 

**Special notes for your reviewer**:
As noted in my comments on #96798, I think the tests here need deep revision in order to become good tests; this PR is a tactical fix to stop the flakes.

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by adtac** [comment]
your four commits LGTM, do you want to rebase on top of Min's latest commit? (I think Min has incorporated your review, so one of your commits might be redundant when you rebase)

**Comment by MikeSpreitzer** [comment]
The force-push to e7b0f582d5d is a rebase onto #96798

**Comment by adtac** [comment]
/lgtm
/retest
(retest to see if we can induce a flake)

**Comment by MikeSpreitzer** [comment]
BTW, `/retest` only re-runs tests whose last run failed.

**Review by yue9944882 — COMMENTED** [review]
/lgtm

File: test/e2e/apimachinery/flowcontrol.go [hunk 1]
```diff
@@ -150,28 +154,31 @@ var _ = SIGDescribe("API priority and fairness", func() {
...
+			concurrencyMultiplier       float64
+			concurrency                 int32
+			completedRequests           int32
+			expectedCompletedPercentage float64
 		}
 		clients := []client{
-			{username: "highqps", qps: 100.0, concurrencyMultiplier: 2.0},
-			{username: "lowqps", qps: 5.0, concurrencyMultiplier: 0.5},
+			{username: highQPSClientName, qps: 90, concurrencyMultiplier: 2.0, expectedCompletedPercentage: 0.90},
+			{username: lowQPSClientName, qps: 4, concurrencyMultiplier: 0.5, expectedCompletedPercentage: 0.90},
```
Comment (lavalamp) [inline test/e2e/apimachinery/flowcontrol.go:181]: Isn't this .9 an increase over #96798's .75?

**Comment by lavalamp** [comment]
/approve

Multiple reviewers seem to think this will help so let's merge it--I don't quite understand but I don't want to block this.

File: test/e2e/apimachinery/flowcontrol.go (hunk 1 above)
Comment (lavalamp) [reply test/e2e/apimachinery/flowcontrol.go:181]: A recent failure:

`Dec  4 15:38:28.824: client "highqps-apf-8126": got 36.6% completed requests, want at least 75.0%`

So I think we'll need a followup.

Perhaps we need to use quite large share amounts to throttle the rest of the cluster.

**Comment by bai** [comment]
> /milestone v1.20

Hey there, just wanted to mention that this was after test freeze and the branch cut. If you’d like this in 1.20, it needs to be cherry picked and the release is tomorrow so this probably won’t make it until 1.20.1.

**Comment by adtac** [comment]
The tests have been disabled in 1.20 (https://github.com/kubernetes/kubernetes/pull/97075), so I don't think it matters too much whether or not we cherry-pick this.

**Pull request merged**